apps/core/scenes/search/PlacePicker/PlacePickerContent.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+// TODO this is just the temporary Picker will be replaced by the OptionPicker form the 'universal-components'
+
+import { graphql, createRefetchContainer } from '@kiwicom/margarita-relay';
+import { type RelayRefetchProp } from 'react-relay'; // eslint-disable-line no-restricted-imports
+import * as React from 'react';
+import { View, ScrollView } from 'react-native';
+import {
+  Text,
+  TextInput,
+  Icon,
+  StyleSheet,
+} from '@kiwicom/universal-components';
+
+import type { PlacePickerContent_locations as PlacePickerContentType } from './__generated__/PlacePickerContent_locations.graphql.js';
+
+type Props = {|
+  +locations: ?PlacePickerContentType,
+  +type: string,
+  +relay: RelayRefetchProp,
+|};
+
+type State = {|
+  text: string,
+|};
+
+class PlacePicker extends React.Component<Props, State> {
+  state = {
+    text: '',
+  };
+
+  handleChangeText = (text: string) => {
+    this.setState({ text });
+    this.props.relay.refetch({ input: { term: text } });
```

Comment:
Yes please! I believe there is a debounce function in some kiwi utility package.

Reply:
Sure I have plan in next PR use `throttle` is better `debounce` for that?

Reply:
OK `debounce` is what we want to.
https://css-tricks.com/debouncing-throttling-explained-examples/

Reply:
@RobinCsl I will check thanks for the idea, anyway I would like to add this feature in next PR.

Reply:
I have been Looked on the gitlab into the incubator but I can't find any `debounce` there.

Reply:
It is not that hard, [example](https://github.com/kiwicom/mobile/blob/198c90b7686b55191d3820d338e44ca136421f83/app/core/src/screens/hotelsStack/placepicker/PlacepickerModal.js#L44) of how it can be done with lodash

Reply:
I know I have experience with the `lodash`, I will add this in my next PR, where a user can finally pick-up a place.